.github/actions/godot-install/action.yml

```diff
@@ -56,7 +56,7 @@ runs:
           chmod 770 "$DIR"
         fi
 
-        DOWNLOAD_URL=https://downloads.tuxfamily.org/godotengine/${{ inputs.godot-version }}
+        DOWNLOAD_URL=https://github.com/godotengine/godot/releases/download/${{ inputs.godot-version }}-${{ inputs.godot-status-version }}
```

Comment:
Thanks, I received this information yesterday too ;)
This change will only work for v4.1.1, the older versions have not been moved to GitHub yet as far as I know.
So this change would currently cause errors when downloading older versions.

Just for info, the download speed will not really increase, because the Godot versions will be cached after the first successful download.

Reply:
update:
i see the older version are now also available on GitHub 👍

Reply:
I thought they were always there? I have been manually downloading builds for my desktop use from GH since forever because TuxFamily was so unreliable and slow.

Anyways, great that worked out now!